### PR TITLE
Update sphinx documentation

### PR DESCRIPTION
**Pull request by NikAzanov**
Improve documentation structure, sphinx configuration and make scripts to build docs

This is not the final state of docs because there are still some changes required for package. I decided to prepare docs structure, then make changes and update docs accordingly

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -34,19 +34,115 @@ adding the following features:
...
-
-Admin migration
-^^^^^^^^^^^^^^^
-Change ``ImportMixin``, ``ExportMixin``, ``ImportExportMixin``
-to ``CeleryImportMixin``, ``CeleryExportMixin`` or ``CeleryImportExportMixin`` respectively.
-
-License
--------
-* Free software: MIT license
+Usage
```
Comment (ron8mcr) [inline README.rst:37]: What do you think about splitting this section into "Installation" (until `collectstatic`) and "Usage"?

File: README.rst [hunk 2]
```diff
@@ -34,19 +34,115 @@ adding the following features:
...
+-----
+
+To install django-import-export-extensions, run this command in your terminal:
```
Comment (ron8mcr) [inline README.rst:40]:
```suggestion
To install `django-import-export-extensions`, run this command in your terminal:
```

File: README.rst [hunk 3]
```diff
@@ -34,19 +34,115 @@ adding the following features:
...
+.. code-block:: python
+
+    # apps/books/resources.py
+    from import_export_extensions.resources import CeleryModelResource
+
+    from .. import models
+
+
+    class BookResource(CeleryModelResource):
+        """Base resource for `Book` model."""
```
Comment (ron8mcr) [inline README.rst:75]:
```suggestion
```

File: README.rst [hunk 4]
```diff
@@ -34,19 +34,115 @@ adding the following features:
...
+    from django.contrib import admin
+
+    from import_export_extensions.admin import CeleryImportExportMixin
+
+    from .. import resources
+
+
+    @admin.register(models.Book)
+    class BookAdmin(CeleryImportExportMixin, admin.ModelAdmin):
+        """Admin UI for Book model."""
```
Comment (ron8mcr) [inline README.rst:95]:
```suggestion
```

File: README.rst [hunk 5]
```diff
@@ -34,19 +34,115 @@ adding the following features:
...
+        resource_class = resources.BookResource
+
+
+Prepare view sets to import/export via API
+
+.. code-block:: python
+
+    # apps/books/api/views.py
+    from ..resources import BookResource
```
Comment (ron8mcr) [inline README.rst:104]: In the previous code sample used `from .. import resources`, let's make consistent

File: README.rst [hunk 6]
```diff
@@ -34,19 +34,115 @@ adding the following features:
...
+
+    class BookImportViewSet(views.ImportJobViewSet):
+        """Simple ViewSet for importing `Book` model."""
+        resource_class = BookResource
+
+
+Don't forget to `configure Celery <https://docs.celeryq.dev/en/latest/getting-started/first-steps-with-celery.html>`_
+if you want to run import/export in background
+
+.. code-block:: python
```
Comment (ron8mcr) [inline README.rst:122]: I think this code sample is redundant in docs of `django-import-export-extensions`.
Also, [celery docs](https://docs.celeryq.dev/en/stable/django/first-steps-with-django.html) provide another code samples for configuring it

File: README.rst [hunk 7]
```diff
@@ -34,19 +34,115 @@ adding the following features:
...
+    app.config_from_object("django.conf:settings", namespace="CELERY")
+
+    app.autodiscover_tasks()
+
+
+Links:
+------
 * Documentation: https://django-import-export-extensions.readthedocs.io.
+* GitHub: https://github.com/saritasa-nest/django-import-export-extensions/
+* Free software: MIT license
```
Comment (ron8mcr) [inline README.rst:147]: This does not look like a link (header of this section is "Links")

File: docs/index.rst [hunk 8]
```diff
@@ -1,17 +1,43 @@
-Welcome to django-import-export-extensions documentation!
-=========================================================
+===============================
+Django import/export extensions
+===============================
+
+django-import-export-extensions is a Django application and library based on
+`django-import-export` library that provided extended features.
+
+**Features:**
```
Comment (ron8mcr) [inline docs/index.rst:8]: Could you add links to docs of each feature?

File: docs/api_drf.rst
Comment (ron8mcr) [inline docs/api_drf.rst:1]: Let's add some prefix for docs (or move them to separate folder) that contains just autogenerated docs

Comment (NikAzanov) [reply docs/api_drf.rst:1]: But they already have `api_` prefix

**Review by ron8mcr — APPROVED** [review]

**Pull request merged**